test/framework/envtest/cluster/finder/env.go

```diff
@@ -0,0 +1,62 @@
+// +build test
+
+package finder
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type EnvFinder struct {
+	prefix             string
+	punctuationPattern *regexp.Regexp
+}
+
+func (f *EnvFinder) Get(camelEnvName string, defaultVal string) string {
+	var envName = f.punctuationPattern.ReplaceAllStringFunc(camelEnvName, func(s string) string {
```

Comment:
I am not sure you are going to convert a camel case string to snake case. If so, I think we can use `https://github.com/iancoleman/strcase` like https://play.golang.org/p/_-HChDbi2Vd

Reply:
Updated